# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kexec-tools/hotfix-error_msg.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- kexec-tools-2.0.25/kexec/arch/x86_64/kexec-elf-x86_64.c.vanilla	2022-12-16 17:35:31.161946394 +0100
+++ kexec-tools-2.0.25/kexec/arch/x86_64/kexec-elf-x86_64.c	2022-12-16 17:36:26.530941485 +0100
@@ -183,7 +183,7 @@
 		elf_rel_set_symbol(&info->rhdr, "entry64_regs", &regs, sizeof(regs));
 
 		if (ramdisk) {
-			error_msg = "Ramdisks not supported with generic elf arguments";
+			error_msg = "Ramdisks not supported with generic elf arguments\n";
 			goto out;
 		}
 	}
--- kexec-tools-2.0.25/kexec/arch/i386/kexec-elf-x86.c.vanilla	2022-12-16 17:37:08.294937781 +0100
+++ kexec-tools-2.0.25/kexec/arch/i386/kexec-elf-x86.c	2022-12-16 17:37:42.133934781 +0100
@@ -248,7 +248,7 @@
 		elf_rel_set_symbol(&info->rhdr, "entry32_regs", &regs, sizeof(regs));
 
 		if (ramdisk) {
-			error_msg = "Ramdisks not supported with generic elf arguments";
+			error_msg = "Ramdisks not supported with generic elf arguments\n";
 			goto out;
 		}
 	}
